app/gui2/e2e/selectingNodes.spec.ts

```diff
@@ -11,9 +11,16 @@ test('Selecting nodes by click', async ({ page }) => {
   await expect(node1).not.toBeSelected()
   await expect(node2).not.toBeSelected()
 
+  const deselectAll = async () => {
+    await page.keyboard.press('Escape')
+    await expect(node1).not.toBeSelected()
+    await expect(node2).not.toBeSelected()
+  }
+
   await locate.graphNodeIcon(node1).click()
   await expect(node1).toBeSelected()
   await expect(node2).not.toBeSelected()
+  await deselectAll()
```

Comment:
This line forfeits one of the covered cases: that clicking at non-selected node also deselect previous one.